src/experimental-layers/package.json

```diff
@@ -61,6 +61,7 @@
     "eslint-config-uber-es2015": "^3.0.0",
     "eslint-config-uber-jsx": "^3.0.0",
     "eslint-plugin-react": "~6.7.0",
+    "deck.gl": "5.x",
```

Comment:
Is 5.x valid version format?

Reply:
> Is 5.x valid version format?

At least for peer dependencies, as you can see below it is standard syntax for React peer dependencies. It appears to work, will keep for now.